pkg/apiserver/rest/usecase/project.go

```diff
@@ -466,6 +471,40 @@ func (p *projectUsecaseImpl) UpdateProjectUser(ctx context.Context, projectName
 	return ConvertProjectUserModel2Base(&projectUser), nil
 }
 
+func (p *projectUsecaseImpl) GetConfigs(ctx context.Context, projectName, configType string) ([]*apisv1.Config, error) {
+	return getConfigsByProjectAndConfigType(ctx, p.k8sClient, projectName, configType)
+}
+
+func getConfigsByProjectAndConfigType(ctx context.Context, k8sClient client.Client, projectName, configType string) ([]*apisv1.Config, error) {
+	if configType != types.TerraformProvider {
```

Comment:
If only query TF providers, this function name should be getProjectTFProvider.
and the api path should be /api/v1/projects/xxx/tf_providers

Reply:
We need to query the image registries list and chart repositories later.